libraries/hatch-server-user-management/src/UserManagementController.ts

```diff
@@ -0,0 +1,208 @@
+import {controller, route} from '@launchtray/hatch-server';
+import {BasicRouteParams} from '@launchtray/hatch-server-middleware';
+import {inject, Logger} from '@launchtray/hatch-util';
+import 'cross-fetch/polyfill';
+import {AUTH_ACCESS_TOKEN_COOKIE_NAME} from './constants';
+import AWSCognitoClient from './AWSCognitoClient';
```

Comment:
Seems like we should make this independent of AWS -- we even call it userService below. Let's create an interface for it.

Reply:
Updated